eq-author/src/components/RichTextEditor/index.js

```diff
@@ -296,8 +317,34 @@ class RichTextEditor extends React.Component {
         replacePipedValues(labels, deletedPlaceholder),
         contentState
       );
-
+      
+    const createNewEntryForMultipleValueEntities = answers => {
+      const processedEntries = [];
+
+      answers.forEach( answer => {
+        
+        if (answer.type === "DateRange") {
+          processedEntries.push({
+            ...answer,
+            id: `${answer.id}from`
+          });
+
+          processedEntries.push({
```

Comment:
I think push can take n arguments so I believe you can just call it once here with both new objects.